gradio/interface.py

```diff
@@ -702,7 +708,9 @@ def attach_submit_events(
                     preprocess=not (self.api_mode),
                     postprocess=not (self.api_mode),
                     show_progress="hidden" if streaming_event else self.show_progress,
```

Comment:
Minor point unrelated to this PR, but I think we should just make show_progress="hidden" if `live=True`